### PR TITLE
Add CI using Travis

### LINKED ISSUE #1: Tests are broken
Due to python 2.7.6 security issues on Mac the tests fail.

### PR DESCRIPTION
**Pull request by sttts**
Fixes #1

### CODE REVIEW

File: tests/fake-kubernetes/dcos.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[core]
+dcos_url = "http://127.0.0.1:8000/"
+reporting = true
+token = "d59d4387b1289bc0f3e5f62740e62e7e6b6df36395d1787a592f1af1a94b745f"
```
Comment (s-urbaniak) [inline tests/fake-kubernetes/dcos.toml]: is this a real token? if so, i'd remove it.

File: dcos_kubernetes/cli.py [hunk 2]
```diff
@@ -111,9 +115,9 @@ def main():
 
     # check whether kubectl binary exists and download if not
     try:
-       from urlparse import urljoin # python 2
+        from urlparse import urljoin  # python 2
```
Comment (s-urbaniak) [inline dcos_kubernetes/cli.py]: are these two additional whitespaces necessary?

File: dcos_kubernetes/cli.py [hunk 3]
```diff
@@ -111,9 +115,9 @@ def main():
...
     except ImportError:
-        from urllib.parse import urljoin # python 3
+        from urllib.parse import urljoin  # python 3
```
Comment (s-urbaniak) [inline dcos_kubernetes/cli.py]: same here, I guess we can remove this little whitespace?!

File: tests/fake-kubernetes/dcos.toml (hunk 1 above)
Comment (sttts) [inline tests/fake-kubernetes/dcos.toml]: done

File: dcos_kubernetes/cli.py (hunk 2 above)
Comment (sttts) [inline dcos_kubernetes/cli.py]: yes

**Comment by s-urbaniak** [comment]
small nits, else :+1: for the travis integration! :-)

File: dcos_kubernetes/cli.py (hunk 3 above)
Comment (sttts) [inline dcos_kubernetes/cli.py]: no

**Pull request merged**